packages/frontend-api/src/Model/Resolver/Products/ProductResolverMap.php

```diff
@@ -84,7 +91,8 @@ public function __construct(
         ?ProductAccessoryFacade $productAccessoryFacade = null,
         ?CurrentCustomerUser $currentCustomerUser = null,
         ?ProductFacade $productFacade = null,
-        ?ProductCachedAttributesFacade $productCachedAttributesFacade = null
+        ?ProductCachedAttributesFacade $productCachedAttributesFacade = null,
```

Comment:
This dependency should not be necessary if I am not missing anything.

Reply:
fixed